openpype/hosts/houdini/api/lib.py

```diff
@@ -127,6 +127,8 @@ def get_output_parameter(node):
         return node.parm("filename")
     elif node_type == "comp":
         return node.parm("copoutput")
+    elif node_type == "opengl":
```

Comment:
The Arnold if branch has another nested if branch in it because it can either render or export ass files and then uses a different filename parameter. So this might not be as trivial for some to be a dict mapping.